MPF.Core/Data/Drive.cs

```diff
@@ -66,10 +68,21 @@ public class Drive
                 string? volumeLabel = Template.DiscNotDetected;
                 if (this.MarkedActive)
                 {
-                    if (string.IsNullOrEmpty(this.VolumeLabel))
-                        volumeLabel = "track";
-                    else
+                    if (!string.IsNullOrEmpty(this.VolumeLabel))
```

Comment:
Since this `if` statement is now doing more heavy lifting, please have it so that the `if (this.MarkedActive)` check is inverted and just returns `volumeLabel` immediately.